dt-impact-tracer/src/edu/washington/cs/dt/impact/Main/ResultsParser.java

```diff
@@ -1,17 +1,496 @@
+                            }
+                        }
+                    }
+
+                } // prioritization techinque, figure 17
+                else if (techniqueName.equals("prioritization")) {
+                    String apfd_value_string = parseFile(file, APFD_VALUE);
+                    double apfd_value = Double.parseDouble(apfd_value_string);
+                    currProj.useFig17();
+                    double[] fig17_array = currProj.get_fig17_values();
+                    // original values, index should be i=0, i+=2
+                    if (resolveDependences == null) {
+                        if (coverageName.equals("statement")) {
+                            if (orderName.equals("absolute")) {
+                                // T3
+                                fig17_array[0] = apfd_value;
+                            } else { // relative
+                                // T4
+                                fig17_array[2] = apfd_value;
+                            }
+
+                        } else if (coverageName.equals("function")) {
+                            if (orderName.equals("absolute")) {
+                                // T5
+                                fig17_array[4] = apfd_value;
+
+                            } else { // relative
+                                // T7
+                                fig17_array[6] = apfd_value;
+                            }
+                        }
+
+                    } else { // auto, index should be i = 1, i+=2
+                        if (coverageName.equals("statement")) {
+                            if (orderName.equals("absolute")) {
+                                // T3
+                                fig17_array[1] = apfd_value;
+                            } else {
+                                // T4
+                                fig17_array[3] = apfd_value;
+                            }
+
+                        } else if (coverageName.equals("function")) {
+                            if (orderName.equals("absolute")) {
+                                // T5
+                                fig17_array[5] = apfd_value;
+
+                            } else {
+                                // T7
+                                fig17_array[7] = apfd_value;
+                            }
+                        }
+
+                    }
+
+                } else {// garbage value...return error
+                    // TODO throw an exception and exit
+                }
+
+            }
+        }
+
+        // generate LaTeX for the orig-written and automatic test suites
+        String origLatexString = generateLatexString(proj_orig_arrayList);
+        String autoLatexString = generateLatexString(proj_auto_arrayList);
+
+        String origOutputFilename = outputDirectoryName + "/";
+        String autoOutputFilename = outputDirectoryName + "/";
+        if (proj_orig_arrayList.get(0).isFig17()) {
```

Comment:
It seems that if a project is true for isFig17 but is also true for isFig18 and isFig19 then only figure 17 results will be outputted.
This should be changed such that *all* figures that is true for a project will be outputted.